packages/components/modules/notifications/native/NotificationsList/styles.ts

```diff
@@ -0,0 +1,20 @@
+import { Theme } from '@baseapp-frontend/design-system/styles/native'
+
+import { StyleSheet } from 'react-native'
+
+export const createStyles = (theme: Theme) =>
+  StyleSheet.create({
+    headerContainer: {
```

Comment:
let's reuse the Appbar styling so it's similar and we have smoother transition (e.g. from the settings page we go to notifications page)
```suggestion
    headerContainer: {
      alignItems: 'center',
      borderBottomColor: theme.colors.surface.border,
      borderBottomWidth: 1,
      flexDirection: 'row',
      height: 64,
      justifyContent: 'space-between',
      padding: 16,
    },
```

I would keep the padding left 16 instead of 20 (that is design) to keep it more consistent. Can you ask TS or DF if we can do it?